sms_sponsorship/forms/sms_sponsorship_registration_form.py

```diff
@@ -162,7 +162,7 @@ def _form_validate_alpha_field(self, field, value):
 
         # Form submission
         #################
-        def form_before_create_or_update(self, values, extra_values):
+        def form_before_create_or_update(self, *values, **extra_values):
```

Comment:
Why did you change that?

Reply:
I didn't change that in my commit, must have been a previous error